robottelo/constants/__init__.py

```diff
@@ -324,8 +324,6 @@
     'immediate': "Immediate",
 }
 
-CHECKSUM_TYPE = {'default': "Default", 'sha256': "sha256", 'sha1': "sha1"}
```

Comment:
I didn't really see value in having this dictionary constant, given its scope of use.